pkg/cli/admin/inspect/util.go

```diff
@@ -191,7 +191,8 @@ func getAllEventsRecursive(rootDir string) (*corev1.EventList, error) {
 			}
 			events, err := readEvents(eventBytes)
 			if err != nil {
-				return fmt.Errorf("failed to read event err: %v", err)
+				klog.Warningf("skipping %s, failed to read event err: %v", string(eventBytes), err)
```

Comment:
`getAllEventsRecursive` should be renamed then to reflect the fact the function will do best-effort instead. E.g. `getEventsRecursive` + mentioning the best-effort intention in the function description.

Reply:
Thanks for review @ingvagabund. You are right, I renamed function.